Examples/ObjectiveC/CacheManagementExample.m

```diff
@@ -0,0 +1,166 @@
+#import "CacheManagementExample.h"
+@import Mapbox;
+
+NSString *const MBXExampleCacheManagement = @"CacheManagementExample";
+
+@interface CacheManagementExample () <MGLMapViewDelegate>
+
+@property (nonatomic) MGLMapView *mapView;
+@end
+
+@implementation CacheManagementExample
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+
+    /* Set the maximum ambient cache size in bytes. Call this method before the map view is loaded.
+
+     The ambient cache is created through the end user loading and using a map view. */
+    NSUInteger maximumCacheSizeInBytes = 62914560;
```

Comment:
Sorry that I wasn’t more clear in my earlier review — the comment above this already states that it’s in bytes, but I don’t want to have to do the conversion of bytes-to-megabytes myself every time I read this or want to edit it.

Two concrete suggestions:
- Use math instead, which offers discrete units that are easier to read/edit.
- Directly comment after the definition.

```suggestion
    NSUInteger maximumCacheSizeInBytes = 64 * 1024 * 1024; // 64 megabytes
```